source/sass/abstracts/_variables.scss

```diff
@@ -68,11 +68,13 @@ $font-sizes: (
 $breakpoints: (
   "mob-portrait" "(min-width: 320px)",
   "mob-landscape" "(min-width: 480px)",
+  // Smaller tablet breakpoint for when using the FRU form
+    "tablet-fundraiseup" "(min-width: 600px)",
   "tablet-portrait" "(min-width: 768px)",
   "tablet-landscape" "(min-width: 1024px)",
-  "paypal-specific" "(min-width: 1090px)",
   // needed for working with responsive paypal button
-    "desktop" "(min-width: 1280px)",
+    "paypal-specific" "(min-width: 1090px)",
```

Comment:
```suggestion
  "paypal-specific" "(min-width: 1090px)",
```

Reply:
Hi @mmmavis! Interestingly, this file already had the "paypal-specific" breakpoint indented after its comment. 🤔

If I un-indent it, the node CI fails due to prettier saying 'please insert ".."' on the lines after the comments.

Reply:
Ah okay. Let's just keep this as is then.